.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Welcome - thanks for contributing to the Grid image management system
+
+We follow the standard GitHub [fork & pull](https://help.github.com/articles/using-pull-requests/#fork--pull) approach to pull requests. i.e. Fork upstream, develop in a branch, and submit a PR.
+
+If you're not sure how to approach a contribution you want to make then feel free to open an issue instead and we'd be glad to discuss it with you.
+
+## Contribution process
+
+1. To avoid wasted effort it is often worth talking to the grid devs to discuss what you are hoping to do, you never  know - we might already have implemented what you need! Feel free to open an issue first and we'd be glad to figure that out with you.
```

Comment:
Shoul _grid_ always be capitalised? Two spaces between _never  know_.

Reply:
Have capitalised. Spacing doesn't much matter in MD as it won't appear in the rendered version (same as HTML). That said I've removed it 😄